package.json

```diff
@@ -16,73 +16,73 @@
     "test": "cross-env TZ=UTC jest",
     "test:watch": "yarn run test --watch"
   },
+  "engines": {
+    "node": "^18.0.0",
```

Comment:
Should we add this to `/.nvmrc` as well?

Reply:
Some tools look at the .nvmrc (like Netlify), but I’d prefer this alongside the project dependencies here because those _are_ effectively project dependencies. Tools like npm, yarn, and friends will check against values in `engines`, too, so it’s good to have it here.